ansibullbot/triagers/plugins/component_matching.py

```diff
@@ -1,6 +1,34 @@
 import logging
 import re
 
+ACTION_PLUGIN_PATTERN = re.compile(r'(?:lib/ansible|plugins)/action')
+MODULE_PATTERN = re.compile(r'(?:lib/ansible|plugins)/modules')
+MODULE_UTIL_PATTERN = re.compile(r'(?:lib/ansible|plugins)/module_utils')
+
+# Known possible Ansible plugin types
+PLUGIN_TYPES = [
```

Comment:
We do not seem to differentiate between plugins why is it needed to hardcode them? To filter "bogus" directories in `plugins/`?

Reply:
Good question, I'm trying to think of something and coming up short :rofl:

It seemed like the bot was interested in determining different plugin types but it wasn't catching things outside of the few directories that were previously hardcoded so the intent is to make it a bit more generic.

Another challenge was to map the slightly different paths between ansible/ansible and inside collections.